### PR TITLE
🔨 Use laravel route helper function instead hardcode the uri.

### PR DESCRIPTION
**Pull request by UnknownRori**
### Purpose of change.

Since some of the `route` are defined using `Route::resource` these also generate not only typical `http` request but also generate name for the individual `route`, so we can remove all hardcode `uri` in the link and form submit action to use Laravel helper method route to generate the `uri` we already define in route.

## Describe the solution.

Replace all hardcode link and form submit action to use function `route`, and added `name prefix` for unamed `route`.

### Describe alternatives you've considered.

Leave as is, or maybe you want to implement in your own naming convention, you are feel free to do so.

### Testing.

Working properly without any error in local server.

### Reference

[Laravel route name prefix docs.](https://laravel.com/docs/9.x/routing#route-group-name-prefixes)
[Laravel route resource action docs.](https://laravel.com/docs/9.x/controllers#actions-handled-by-resource-controller)
[Laravel route helper function docs.](https://laravel.com/docs/9.x/helpers#method-route)

### CODE REVIEW

File: resources/views/components/header.blade.php [hunk 1]
```diff
@@ -1,27 +1,27 @@
 <nav class="navbar navbar-expand-lg navbar-dark bg-success">
     <div class="container">
-        <a class="navbar-brand" href="/">Warnetku</a>
+        <a class="navbar-brand" href="route('home')">Warnetku</a>
```
Comment (BayuDC) [inline resources/views/components/header.blade.php:3]: Why this one is not inside ```{{ }}```?

File: resources/views/computer/index.blade.php [hunk 2]
```diff
@@ -25,7 +25,7 @@
     @endforeach
     @can('is-owner')
     <div class="col-lg-3 col-md-4 col-sm-6 col-12">
-        <a href="/computer/create" class="p-4 bg-light border border-success d-block text-success h-100">
+        <a href="route('computer.create')" class="p-4 bg-light border border-success d-block text-success h-100">
```
Comment (BayuDC) [inline resources/views/computer/index.blade.php:28]: and this

**Review by BayuDC — CHANGES_REQUESTED** [review]

File: resources/views/components/header.blade.php (hunk 1 above)
Comment (UnknownRori) [reply resources/views/components/header.blade.php]: sorry seem like i forgot to add it, i am going to fix now.

**Review by BayuDC — APPROVED** [review]
Looks good

**Comment by BayuDC** [comment]
Thanks for the contribution, it really helped me. 

And btw, do you have a Discord account?

**Comment by UnknownRori** [comment]
Yes, why you ask?

**Comment by BayuDC** [comment]
Let's discuss more about this project

**Comment by UnknownRori** [comment]
All right, I will send it via whatsapp so please be prepare for unknown number.

**Pull request merged**